instrumentation/rxjava-2.0/library/src/main/java/io/opentelemetry/instrumentation/rxjava2/TracingCompletableObserver.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.rxjava2;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.reactivex.CompletableObserver;
+import io.reactivex.annotations.NonNull;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.internal.disposables.DisposableHelper;
+
+class TracingCompletableObserver implements CompletableObserver, Disposable {
+
+  private final CompletableObserver actual;
+  private final Context parentSpan;
```

Comment:
```suggestion
  private final Context context;
```

Can you rename this throughout?

Reply:
sure, will do